packages/shared/src/components/cards/CollectionCard/CollectionCard.tsx

```diff
@@ -30,7 +29,7 @@ export const CollectionCard = forwardRef(function CollectionCard(
   }: PostCardProps,
   ref: Ref<HTMLElement>,
 ) {
-  const [hovered, toggleHovered] = React.useState(false);
+  const [isHovered, setHovered] = useState(false);
```

Comment:
I don't think we need the state, see my later comment on using css hover (previously we needed it for the loading on demand, but since we no longer use that, css could work.

Reply:
You are right, CSS to the rescue 👌 and also more performant!